docs/docsite/rst/contributor/index.rst

```diff
@@ -10,8 +10,7 @@ There are so many ways you can contribute to AWX.
 
 **Join us online**
 
-You can chat with us and ask questions on Matrix at `#awx:ansible.com <https://matrix.to/#/#awx:ansible.com>`_.
-Also visit the `forum.ansible.com <https://forum.ansible.com/>`_ to find contributor resources.
+You can chat with us and ask questions on Matrix at `#awx:ansible.com <https://matrix.to/#/#awx:ansible.com>`_ or visit the `Ansible Community Forum <https://forum.ansible.com/c/project/7/>`_ to find contributor resources.
```

Comment:
Overall looks good but I think "Ansible Forum" is the way to go over "Ansible Community Forum" (drop the community). Not to say that it isn't a community thing I just think that would be consistent so nobody thinks there are two forums.

Reply:
I stand corrected here. I checked with the team and it actually should be "Ansible Community Forum" to keep the separation clear. So disregard me. Thanks.

Reply:
I second this.... I got the direction from @leogallego.